cdserver/config/initializers/app_metadata.rb

```diff
@@ -0,0 +1,10 @@
+module MySite
+  class Application
+    NAME = 'capdash2'
+    VERSION = '0.0.1'
+    DESCRIPTION = "Shelter Capacity Dashboard"
+    RAILS_VERSION = Rails.version
```

Comment:
Actually I was thinking the rails stuff should still be in the controller.  Rails.version is already a constant, so no need to duplicate that in app_metadata.rb    app_metadata.rb should only be the app-specific metadata which is only the three NAME, VERSION, and DESCRIPTION.   All others should remain the way you originally coded it.  Sorry for confusion